containers/src/Kokkos_DualView.hpp

```diff
@@ -86,22 +86,37 @@ inline const Kokkos::Cuda& get_cuda_space(const NonCudaExecSpace&) {
 #endif  // KOKKOS_ENABLE_CUDA
 
 }  // namespace Impl
-template <class DataType, class Arg1Type = void, class Arg2Type = void,
-          class Arg3Type = void>
-class DualView : public ViewTraits<DataType, Arg1Type, Arg2Type, Arg3Type> {
-  template <class, class, class, class>
+
+template <class DataType, class... Properties>
+class DualView;
+
+template <class>
+struct is_dual_view : public std::false_type {};
+
+template <class D, class... P>
+struct is_dual_view<DualView<D, P...>> : public std::true_type {};
+
+template <class D, class... P>
+struct is_dual_view<const DualView<D, P...>> : public std::true_type {};
```

Comment:
Somewhat unrelated change that is not captured in the PR description.
Must clearly state that is being added and why (consistency with other containers) so that it can be easily tracked.

Reply:
At the beginning I didn't introduced `is_dual_view`. But in `struct DualViewSubviewType` it was somehow needed to ensure that `typename DV` is a dual view and not something else.

I'll update the description of the PR accordingly for sure!